RecoMuon/DetLayers/src/MuonGEMDetLayerGeometryBuilder.cc

```diff
@@ -49,6 +65,68 @@ pair<vector<DetLayer*>, vector<DetLayer*> > MuonGEMDetLayerGeometryBuilder::buil
   return res_pair;
 }
 
+MuRingForwardLayer* MuonGEMDetLayerGeometryBuilder::buildLayer0(int endcap,
```

Comment:
If I am not mistaken, `buildLayer0` is identical to `buildLayer` and there is no need to duplicate them: the only difference is for `bool isInFront`, for which please see below.
All the comments below apply to `buildLayer` as well